src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -267,6 +267,7 @@
     'asan_check_access',
     'asan_osx_dynamic.dylib',
     'assert',
+    'do_malloc',
```

Comment:
This is for tcmalloc frames like: (anonymous namespace)::do_malloc(unsigned long)

Because of the prefix it has to be in IGNORE_CONTAINS.